index.html

```diff
@@ -213,14 +213,14 @@ <h4>Skills</h4>
       <h2>Contact me</h2>
       <p>If you have an application you are interested in developing, a feature that you need built or a project that
         needs coding. I’d love to help with it </p>
-      <form class="form flex" action="submit" method="post">
-        <div class="input flex">
-          <input name="name" type="text" required placeholder="Name" maxlength="30" />
-          <input name="email address" type="text" required placeholder="Email address" maxlength="30" />
-        </div>
-        <textarea name="message" placeholder="Write your message here" maxlength="500" required></textarea>
-        <button class="last-btn" type="submit">Get It Touch</button>
-      </form>
+        <form class="form flex" action="https://formspree.io/f/xwkzpvdn" method="post">
+          <div class="input flex">
+            <input name="firstname" type="text" placeholder="First Name" maxlength="30" required/>
+          </div>
+          <input name="email" type="text" placeholder="Email address" maxlength="30" required/>
+          <textarea name="message" placeholder="Write your message here" maxlength="500" required></textarea>
+          <button class="last-btn" type="submit">Get It Touch</button>
+        </form>
```

Comment:
- [ ] I think the paddings beside your form inputs are too much when compared to the Figma design template. Please adjust the padding and make sure it matches the Figma design you chose. In real life working environment, you will be expected to follow design guidelines strictly 😄 

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/63473584/186219543-5f321235-345d-47cd-bf68-d9d1d42b8b4b.png)  | ![image](https://user-images.githubusercontent.com/63473584/186227657-6e105906-7f64-443b-a278-9d71908d1345.png)  |